<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DBHOMEHP" ignored="4" name="Sanity Test" tests="1" failures="1" timestamp="2022-12-11T14:53:12 GMT" time="0.051" errors="0">
  <testcase name="bankManagerShouldAddCustomerSuccessfully" time="0.006" classname="com.bank.testsuite.BankTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.bank.testsuite.BankTest.bankManagerShouldAddCustomerSuccessfully(BankTest.java:31)
at java.util.ArrayList.forEach(ArrayList.java:1249)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- bankManagerShouldAddCustomerSuccessfully -->
  <testcase name="customerShouldDepositMoneySuccessfully" time="0.0" classname="com.bank.testsuite.BankTest">
    <ignored/>
  </testcase> <!-- customerShouldDepositMoneySuccessfully -->
  <testcase name="bankManagerShouldOpenAccountSuccessfully" time="0.0" classname="com.bank.testsuite.BankTest">
    <ignored/>
  </testcase> <!-- bankManagerShouldOpenAccountSuccessfully -->
  <testcase name="customerShouldLoginAndLogoutSuccessfully" time="0.0" classname="com.bank.testsuite.BankTest">
    <ignored/>
  </testcase> <!-- customerShouldLoginAndLogoutSuccessfully -->
  <testcase name="customerShouldWithdrawMoneySuccessfully" time="0.0" classname="com.bank.testsuite.BankTest">
    <ignored/>
  </testcase> <!-- customerShouldWithdrawMoneySuccessfully -->
</testsuite> <!-- Sanity Test -->
